### ISSUE TITLE
Which label-set do you use for evaluation?

### ISSUE THREAD

**Issue by glennliu** [y9miao/volumetric-semantically-consistent-3D-panoptic-mapping#1]
Hi 

May I know which label-set you used for mAP evaluation?  The paper mentions 18 semantic classes but I cannot find the exact label-set.
If you use 20 NYUv2 label-set (wall,floor,cabine,bed,bookshelf ...etc), many of them cannot be detected by Mask R-CNN. And Voxblox++ should achieve 0 mAP on many of those classes since Mask R-CNN never detects some of them.

Also, can you also share which detectron2 model weight you use for panoptic segmentation?

Thanks

**Comment by y9miao** [comment]
Hi @glennliu ,

Thanks for your interest in our work. I have update some implementation details in the Readme file which might make it more clear. 

Regarding to your questions, 
1. We use  18 out of 20 NYUv2 label-set without wall and floor.
2. In order to output NYU labels, we finetune the Mask2former model with ScanNet train set from here:
https://github.com/facebookresearch/Mask2Former/blob/main/configs/coco/panoptic-segmentation/swin/maskformer2_swin_base_IN21k_384_bs16_50ep.yaml, and I just upload the finetuned model link.
3. In order for fair comparison, we run voxblox++ with the finetuned mask2former instead of Mask-RCNN.
Hope these answers your question.

**Comment by glennliu** [comment]
> Hi @glennliu ,
> 
> Thanks for your interest in our work. I have update some implementation details in the Readme file which might make it more clear.
> 
> Regarding to your questions,
> 
> 1. We use  18 out of 20 NYUv2 label-set without wall and floor.
> 2. In order to output NYU labels, we finetune the Mask2former model with ScanNet train set from here:
>    https://github.com/facebookresearch/Mask2Former/blob/main/configs/coco/panoptic-segmentation/swin/maskformer2_swin_base_IN21k_384_bs16_50ep.yaml, and I just upload the finetuned model link.
> 3. In order for fair comparison, we run voxblox++ with the finetuned mask2former instead of Mask-RCNN.
>    Hope these answers your question.

Thanks for your response. Nice work!
